src/nvim/ui.h

```diff
@@ -22,6 +23,7 @@ struct ui_t {
   void (*clear)(UI *ui);
   void (*eol_clear)(UI *ui);
   void (*cursor_goto)(UI *ui, int row, int col);
+  void (*cursor_style_set)(UI *ui, Dictionary args); // or a Dictionary ?
```

Comment:
But then it implies that UIs have a definition of the Dictionary type, and that they include nvim/api/private/defs.h, which doesn't seem to be currently the case (looking at you nvim-qt) ?